script/linux

```diff
@@ -21,6 +28,7 @@ if [[ -n $apt ]]; then
   deps=(
     libasound2-dev
     libfontconfig-dev
+    vulkan-validationlayers*
```

Comment:
Are we sure we need this for everyone? It's only required for developers, see https://github.com/zed-industries/zed/pull/7343/files#diff-e6075538b36d33b199cdfd03cfae8988bdcb787d012b369eca2a3607f95b239eR199

Reply:
We can make it configurable before the final merge to only be used during the debug builds.

Reply:
sounds good!